app/views/streams/main_stream.html.haml

```diff
@@ -4,8 +4,8 @@
 
 - content_for :head do
   - if AppConfig.chat.enabled?
-    = javascript_include_tag :jsxc, :id => 'jsxc',
-      :data => { :endpoint => get_bosh_endpoint }
+    = javascript_include_tag :jsxc, id: 'jsxc',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.